tests/components/uptimerobot/test_config_flow.py

```diff
@@ -22,7 +22,6 @@
 
 async def test_form(hass: HomeAssistant) -> None:
     """Test we get the form."""
-    await setup.async_setup_component(hass, "persistent_notification", {})
```

Comment:
I think we should keep this. It avoids an error logged when setting up the integration eg on create entry results.